test/unittests/api_test.cpp

```diff
@@ -7,6 +7,7 @@
 #include <gtest/gtest.h>
 #include <lib/fizzy/limits.hpp>
 #include <test/utils/asserts.hpp>
+#include <test/utils/execute_helpers.hpp>
```

Comment:
Does this file needs it?

Reply:
It has the overload for `execute(..., {...})`.

Reply:
But no new test was added, why wasn't this needed before?

Reply:
Previously, the args were passed by `std::vector` which has constructor for `{}`.